app/uk/gov/hmrc/helptosavefrontend/views/email/give_email.scala.html

```diff
@@ -21,7 +21,7 @@
 @import uk.gov.hmrc.helptosavefrontend.config.FrontendAppConfig
 @import uk.gov.hmrc.helptosavefrontend.views.helpers.errorPrefix
 
-@(form: Form[GiveEmail])(implicit htsContext: HtsContext, request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
+@(form: Form[GiveEmail], backLink: String = "")(implicit htsContext: HtsContext, request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
```

Comment:
is a default argument needed here? A blank string isn't a valid back link

Reply:
otherwise , i have to pass the empty string from inside controller few times, - it may not be valid but clicking on the href this wont do anything